### PR TITLE
chore(documentation): add request documentation

### PR DESCRIPTION
**Pull request by Phil91**
## Description

* add postman collection
* add documentation with example requests

## Why

To have some example requests and explanation for the requests as well as a postman collection to test the policy hub

## Issue

N/A

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/policy-hub/blob/main/docs/developer/Technical-Documentation/Dev-Process/How-to-contribute.md#commit-and-pr-guidelines)
- [x] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [x] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have added tests that prove my changes work
- [x] I have checked that new and existing tests pass locally with my changes
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by evegufy** [comment]
@Phil91 could you please update the PR if needed?

**Comment by Phil91** [comment]
> @Phil91 could you please update the PR if needed?

@evegufy I did double check and rebased to the latest version

File: docs/developer/Technical-Documentation/requests/example-requests.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+```
+
+If you want to try it out, check the [postman collection](/docs/developer/Technical-Documentation/requests/policy-hub.postman_collection.json)
+
+## NOTICE
+
+This work is licensed under the [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+- SPDX-License-Identifier: Apache-2.0
+- SPDX-FileCopyrightText: 2021-2023 Contributors to the Eclipse Foundation
```
Comment (evegufy) [inline docs/developer/Technical-Documentation/requests/example-requests.md:227]: could you please still update the year?

**Review by evegufy — CHANGES_REQUESTED** [review]

File: docs/developer/Technical-Documentation/requests/example-requests.md (hunk 1 above)
Comment (Phil91) [reply docs/developer/Technical-Documentation/requests/example-requests.md]: oh thanks for the hint - updated

**Review by evegufy — APPROVED** [review]

**Pull request merged**